recipe.yaml

```diff
@@ -1,10 +0,0 @@
----
-RecipeFormatVersion: '2020-01-25'
-ComponentName: aws.greengrass.LogManager
-ComponentDescription: AWS Greengrass Log Manager
```

Comment:
Don't commit this change - this was only for testing your changes (revert this file)